lib/routes/binance/announcement.ts

```diff
@@ -0,0 +1,169 @@
+    const baseUrl = 'https://www.binance.com';
+    const announcementCategoryUrl = `${baseUrl}/support/announcement`;
+    const { type } = ctx.req.param<'/binance/announcement/:type'>();
+    const language = ctx.req.header('Accept-Language');
+    const headers = {
+        Referer: baseUrl,
+        'Accept-Language': language ?? 'en-US,en;q=0.9',
+    };
+    const announcementsConfig = (await cache.tryGet(`binance:announcements:${language}`, async () => {
+        const announcementRes = await ofetch<string>(announcementCategoryUrl, { headers, redirect: 'follow' });
+        const $ = cheerio.load(announcementRes);
+
+        const appData = JSON.parse($('#__APP_DATA').text());
+
+        const announcements = Object.values(appData.appState.loader.dataByRouteId as Record<string, object>).find((value) => 'playlist' in value) as { playlist: unknown };
+
+        if (!assertAnnouncementsConfig(announcements.playlist)) {
+            throw new Error('Get announcement config failed');
+        }
+
+        const listConfigProps = announcements.playlist.reactRoot[0].children.find((i) => i.id === 'TopicCardList')?.props;
+
+        if (!assertAnnouncementsConfigList(listConfigProps)) {
+            throw new Error("Can't get announcement config list");
+        }
+
+        return listConfigProps.config.list;
+    })) as AnnouncementsConfig[];
+
+    const announcementCatalogId = ROUTE_PARAMETERS_CATALOGID_MAPPING[type];
+
+    if (!announcementCatalogId) {
+        throw new Error(`${type} is not supported`);
+    }
+
+    const targetItem = announcementsConfig.find((i) => i.url.includes(`c-${announcementCatalogId}`));
+
+    if (!targetItem) {
+        throw new Error('Unexpected announcements config');
+    }
+
+    const link = `${baseUrl}${targetItem.url.replace(baseUrl, '')}`;
+
+    const response = await ofetch<string>(link, { headers, redirect: 'follow' });
+
+    const $ = cheerio.load(response);
+    const appData = JSON.parse($('#__APP_DATA').text());
+
+    const values = Object.values(appData.appState.loader.dataByRouteId as Record<string, object>);
+    const catalogs = values.find((value) => 'catalogs' in value) as { catalogs: AnnouncementCatalog[] };
+    const catalog = catalogs.catalogs.find((catalog) => catalog.catalogId === announcementCatalogId);
+
+    const item = await Promise.all(
+        catalog!.articles.map((i) => {
+            const link = `${announcementCategoryUrl}/${i.code}`;
+            const item = {
+                title: i.title,
+                link,
+                description: i.title,
+                pubDate: parseDate(i.releaseDate),
+            } as DataItem;
+            return cache.tryGet(`binance:announcement:${i.code}:${language}`, async () => {
+                const res = await ofetch(link, { headers, redirect: 'follow' });
+                const $ = cheerio.load(res);
+                item.description = $('#support_article').html() ?? '';
```

Comment:
```suggestion
                item.description = $('#support_article > div').first().html() ?? '';
```
Consider removing part of their self-promotion.